src/main/java/com/leverx/learningmanagementsystem/btp/registry/saas-provisioning.json

```diff
@@ -0,0 +1,8 @@
+{
+  "xsappname": "learning-management-system",
+  "appUrls": {
+    "onSubscription": "https://067769ebtrial-dev-learning-management-system.cfapps.us10-001.hana.ondemand.com/api/v1/subscribe/{tenantId}"
+  },
+  "displayName": "Learning Management System",
```

Comment:
put `appName`, `displayName`, `description`, `category` at top of json (after xsappname), it should play role of title